tensorflow/tools/ci_build/target.lst

```diff
@@ -0,0 +1,2 @@
+gfx803
+gfx900
```

Comment:
how about `gfx906`?

@dagamayank  could you weigh in?

Reply:
Lets hold off for a bit until we have `gfx906` nodes to use for CI. Any preference to add it right now?

Reply:
I initially added `gfx906`, but noticed quite a few compiler code gen warnings.  When will our compiler release add that support?